### ISSUE TITLE
Use friendly device name for DENON receivers

### ISSUE THREAD

**Issue by mkulesh** [mkulesh/onpc#292]
The friendly name can be obtained using "Get Player Info" command:
Command: heos://player/get_player_info?pid=player_id
Response:
```
{"heos": {"command": "player/get_player_info", "result": "success", "message": "pid=-2078441090"}, "payload": {"name": "Denon Player", "pid": -2078441090, "model": "Denon DRA-800H", "version": "3.1.232", "ip": "", "network": "wired", "lineout": 0, "serial": ""}}
```

To change the name:
```
http://xx.xx.xx.xx:8080/goform/formiPhoneAppDirect.xml?NSFRN%20Denon%20Player1
````

**Comment by mkulesh** [comment]
The friendly name is now shown and can be changed in the DEVICE tab
![friendly_name](https://github.com/mkulesh/onpc/assets/30557348/588b4d51-22a2-41eb-a5b4-456dac59049b)

**Comment by mkulesh** [comment]
Released in version 2.13.2

**Comment by mkulesh** [comment]
Released in version 1.32
